### PR TITLE
[currency_rate_update] Adding context as parameter

### PR DESCRIPTION
**Pull request by Assem**

### CODE REVIEW

**Comment by gurneyalex** [comment]
created @Assem profile in the OCA database, but no CLA found.

**Comment by Assem** [comment]
Hello,

I have just downloaded the CLA and signed it, but I don't see how to send it to you!

Thkx for your help,

Assem.

**Comment by Assem** [comment]
I found it and CLA sent

**Comment by max3903** [comment]
I processed the CLA. Problem should be solved.

**Comment by Assem** [comment]
Thkx Max :)

**Comment by yvaucher** [comment]
:+1:

**Comment by StefanRijnhart** [comment]
Thank you. May I ask you to add the context in other places as well? It seems to be missing all over the place, in read/write/browse calls.

File: currency_rate_update/currency_rate_update.py [hunk 1]
```diff
@@ -107,7 +107,7 @@ class Currency_rate_update_service(osv.Model):
         )
     ]
 
-    def _check_max_delta_days(self, cr, uid, ids):
+    def _check_max_delta_days(self, cr, uid, ids, context=None):
         for company in self.read(cr, uid, ids, ['max_delta_days']):
```
Comment (yvaucher) [inline currency_rate_update/currency_rate_update.py:111]: You need to propagate the context here.

**Comment by adrienpeiffer** [comment]
I'll close this one as there is no reaction for 2 years. Feel free to reopen it.

**Pull request closed without merging**